man/src/oxide-config.1.md

```diff
@@ -0,0 +1,268 @@
+% OXIDE-CONFIG(1) oxide-config 0.1.0
+% Philipp Kalinowski
+% February 2023
+
+# NAME
+
+oxide-config - config of Oxide
+
+# DESCRIPTION
+
+Define the behavior of Oxide.
+The config file provides the possibility to customize e. g. keybindings, layout, style.
+If the home config file is not existing, default values will be used but commands like `exec` and `exec_always` will not be working.
+The config file is written in YAML.
+
+# FILES
+
+During launch, Oxide searches for a config file in the following locations:
+
+**~/.config/oxide/config.yml**
+: home config file
+
+**/etc/oxide/config.yml**
+: system config file
+
+# KEYBINDING
+
+## KEYS
+
+A keybinding has to consist of at least one or more MODIFIERS and exactly one normal key such as 't' for example.
+
+## MODIFIER
+
+**M**
+: Meta key
+
+**A**
+: ALT key
+
+**C**
+: CONTROL key
+
+**S**
+: SHIFT key
+
+# COMMANDS
+
+Commands consist of a command and optional arguments.
+
+## COMMAND
+
+Move **args** [MOVEMENT]
+: move window
+
+Focus **args** [MOVEMENT]
+: move focus
+
+Quit
+: quit the window manager
+
+Kill
+: kill the currently focused window
+
+Restart
+: reloads the config and restarts components
+
+Layout **args** [LAYOUT]
+: change the current layout
+
+GoToWorkspace **args** [WORKSPACE_ARGS]
+: change the current workspace
+
+MoveToWorkspace **args** [WORKSPACE_ARGS]
+: move the used window to a different workspace
+
+MoveToWorkspaceAndFollow **args** [WORKSPACE_ARGS]
+: move the focused window to and select a different workspace
+
+Exec **args** COMMAND
+: execute a given command
+
+Fullscreen
+: toggle fullscreen mode for the focused window
+
+## ARGS
+
+Command arguments are necessary for the movement, the layout or to control workspaces.
+
+## MOVEMENT
+
+Left
+: moves to the left
+
+Right
+: moves to the right
+
+## LAYOUT
+
+VerticalStriped
+: windows vertically next to each other
+
+HorizontalStriped
+: windows horizontally underneath each other
+
+None
+: if no argument is provided, the next layout is chosen
+
+## WORKSPACE_ARGS
+
+Next
+: next initialized workspace with a higher index than the current workspace
+
+Previous
+: next initialized workspace with a lower index than the current workspace
+
+Next_free
+: next available workspace with a higher index than the current workspace which is not initialized
```

Comment:
Next
: next initialized workspace with a higher index than the current workspace. If the workspace with the highest index is selected, the index with the lowest index will be selected.

Previous
: next initialized workspace with a lower index than the current workspace. If the workspace with the lowest index is selected, the index with the highest index will be selected.

Next_free
: next available workspace with which is not initialized. Gaps in the workspace indices are filled first.

?:)

Reply:
Good idea, thank you 👍🏼